### PR TITLE
Add FDO support on RHEL 8

### PR DESCRIPTION
**Pull request by shrikant1407**
Signed-off-by: Shrikant Temburwar <shrikant.temburwar@intel.com>

### CODE REVIEW

File: component-samples/demo/scripts/enable_rhel_support.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# This script is used to enable FDO support on REHL 8
```
Comment (DukeDavis12) [inline component-samples/demo/scripts/enable_rhel_support.sh:1]: Please add the copyrights header.

File: component-samples/demo/scripts/enable_rhel_support.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (DukeDavis12) [inline component-samples/demo/scripts/enable_rhel_support.sh:2]: Consider adding a small description of the script.

File: component-samples/demo/rv/Podmanfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Intel Corporation
```
Comment (DukeDavis12) [inline component-samples/demo/rv/Podmanfile:1]: 2021

File: component-samples/demo/reseller/Podmanfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Intel Corporation
```
Comment (DukeDavis12) [inline component-samples/demo/reseller/Podmanfile:1]: 2021

File: component-samples/demo/owner/Podmanfile [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Intel Corporation
```
Comment (DukeDavis12) [inline component-samples/demo/owner/Podmanfile:1]: 2021

File: component-samples/demo/manufacturer/Podmanfile [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Intel Corporation
```
Comment (DukeDavis12) [inline component-samples/demo/manufacturer/Podmanfile:1]: 2021

File: component-samples/demo/aio/Podmanfile [hunk 7]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Intel Corporation
```
Comment (DukeDavis12) [inline component-samples/demo/aio/Podmanfile:1]: 2021

**Review by DukeDavis12 — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**